doc/managing-certificates-for-application-ingress.md

```diff
@@ -0,0 +1,34 @@
+# Managing certificates for application ingress
```

Comment:
Can this be part of CF4K8s install experience? For e.g. we could create a secret resource with a well-known name (as shown in this doc) and use variables for crt/key (from values.yaml). Users can then paste the crt/key in values.yaml at install time.

Reply:
It could be part of the default install for the "default" apps domain... e.g. `*.apps.example.com` that folks want to use -- is that something y'all want? But for each additional domain they want to use I imagine they'd need to do this out of band.